migrations/authgear/20240318191059-add_indexes_to_foreign_key_columns.sql

```diff
@@ -12,6 +12,10 @@ CREATE INDEX _auth_user_role_user_id ON _auth_user_role (user_id);
 CREATE INDEX _auth_user_role_role_id ON _auth_user_role (role_id);
 CREATE INDEX _auth_verified_claim_user_id ON _auth_verified_claim (user_id);
 
+CREATE INDEX _auth_user_app_id ON _auth_user (app_id);
+CREATE INDEX _auth_user_app_id_created_at ON _auth_user (app_id, created_at);
```

Comment:
The default is ASC NULLS LAST. https://www.postgresql.org/docs/current/indexes-ordering.html
But our usage is DESC NULLS LAST.